docs/Transferring-Roms.md

```diff
@@ -11,69 +11,68 @@ There are three main methods of transferring roms: via USB stick, via SFTP, and
-
+8. Remove the USB stick. The ROMs have now been transferred from the USB to the Retropie system's SD card
+9. Refresh the game listing in EmulationStation by pressing F4, or press **Start** on your controller > **Quit** > **Restart EmulationStation**
+10. The transferred games should now be visible within EmulationStation. If any are missing, return to step 6
 
 ### SFTP
 
-[SFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol) or SSH File Transfer Protocol also called Secure File Transfer Protocol is a network protocol that allows you to securely transfer files over a network. Naturally both your PC and RetroPie system will need to be connected to the same network via Ethernet or Wifi in order to successfully transfer your files. 
+SFTP (Secure File Transfer Protocol) is a network protocol that allows you to securely transfer files over the internet or locally on the same network when both your PC and RetroPie system are connected to the same router via Ethernet or Wifi. 
 
 - Wired (needs ethernet cable)
 - Wireless (Raspberry Pi Zero W, 3 and 4 models have onboard Wifi, so Pi 1 and 2 will need a dongle)
 
-To use SFTP, you must first [enable SSH](https://www.raspberrypi.org/documentation/remote-access/ssh/). As of the November 2016 release, Raspbian has the SSH server disabled by default.
-
 To enable SSH from within RetroPie:
 
-1. Select **Configuration/Tools** within **RetroPie Setup**
-2. Select **raspi-config**
-3. Select **Interfacing Options**
-4. Select **SSH**
-5. Choose **Yes**
-6. Select **Ok**
-7. Choose **Finish**
+1. Within the 'RetroPie' EmulationStation options menu, select **raspi-config**
+2. Select **Interface Options**
+3. Select **SSH**
+4. Choose **Yes**
+5. Select **Ok**
+6. Choose **Finish**
 
 There are many SFTP programs out there:
 
-- Windows: [WinSCP](https://winscp.net/eng/download.php)
+- Windows: 
+  - [WinSCP](https://winscp.net/eng/download.php) - For easy drag & drop file transfer
+  - [MobaXTerm](https://mobaxterm.mobatek.net/) - Feature-rich command line access and drag & drop file transfer
 - Mac: [Cyberduck](https://cyberduck.io/?l=en)
 
 ![ftp](https://cloud.githubusercontent.com/assets/10035308/9144892/68994618-3d0d-11e5-8db0-2991f9068115.png)
 
 **Connection settings:** 
 
 - Protocol: `SFTP`
-- IP address: To find the IP address of your RetroPie, go into RetroPie options from the main menu, and select the last option `Show IP address`. You can also find this information from the terminal on retropie in the bash info or with the command `ifconfig`
+- IP address/Host Name: To find the IP address of your RetroPie, go into the 'RetroPie' EmulationStation options menu and select **Show IP**. You can also find this information from the terminal on Retropie in the bash info or with the command `ifconfig`
 - Username: `pi` (default)
 - Password: `raspberry` (default)
 
-**Where to drop the files***
+**Where to drop the files:**
```

Comment:
I'd prefer not to have **:** at the end of a title/heading.

Reply:
Looks like the same line as above. Again, your call.